client/js/render.js

```diff
@@ -195,13 +196,22 @@ function renderChannelUsers(data) {
 }
 
 function renderNetworks(data, singleNetwork) {
+	const collapsed = JSON.parse(storage.get("thelounge.networks.collapsed")) || {};
+
 	sidebar.find(".empty").hide();
 	sidebar.find(".networks").append(
 		templates.network({
 			networks: data.networks,
 		})
 	);
 
+	Object.keys(collapsed)
+		.forEach((key) => {
+			if (collapsed[key]) {
+				collapseNetwork($(`.network[data-uuid=${key}] button.collapse-network`));
```

Comment:
Add quotes around `${key}`, missed that before.

Reply:
> Add quotes around ${key}

Sure. Weight that it worked, I guess it's just being more accepting of stuff

> And button is probably unnecessary in the selector.

Same to what I said to @astorije ... I'd rather keep it in, tbh.